tests/test_pkce.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
[Yes](https://www.youtube.com/watch?v=VX4DJUr5oYg).

By being uniform about the trivialities we free our eyes during code review to catch actually worthwhile problems and concerns. So goes the theory anyway.